### PR TITLE
Tests hotifx

### PR DESCRIPTION
**Pull request by syrcheddar**

### CODE REVIEW

File: modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 class OutputFilesGeneratorTests {
 
     static String actualResult = "";
-    static final String expectedResult = readFileAsString(new File("./src/test/resources/TestingClass_.txt"));
+    static final String expectedResult = readFileAsString(new File("./src/test/resources/TestingClassOWL_.txt"));
```
Comment (ledsoft) [inline modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java:20]: Remove unused constant.

File: modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java [hunk 2]
```diff
@@ -122,48 +128,29 @@ void isAnnotatedWithFailTest() {
...
         @Test
         void containsTypedProperties() {
-            assertTrue(actualResult.contains("public static volatile PropertiesSpecification<TestingClass, Map, URI, Object> propertie;"));
+            assertTrue(actualResult.contains("public static volatile PropertiesSpecification<TestingClassOWL, Test> properties;"));
         }
 
         @Test
         void containsListAttribute() {
-            assertTrue(actualResult.contains("public static volatile ListAttribute<TestingClass, String> listAttribute;"));
+            assertTrue(actualResult.contains("public static volatile ListAttribute<TestingClassOWL, Test> data-list;"));
```
Comment (ledsoft) [inline modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java:148]: This is not a valid Java field name - you cannot use `-` in names.

File: modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java [hunk 3]
```diff
@@ -122,48 +128,29 @@ void isAnnotatedWithFailTest() {
...
-
-
-        String readFileAsString(File file) {
-            try {
-                Scanner scanner = new Scanner(file);
-                String fileContents = scanner.useDelimiter("\\Z").next();
-                scanner.close();
-                return fileContents;
-            } catch (IOException e) {
-                System.err.println("Chyba cteni");
```
Comment (ledsoft) [inline modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java:164]: This catch is useless (in the static method on line 159). Throw a runtime exception wrapping the IO exception or declare the whole method to throw the IOException.

**Review by ledsoft — CHANGES_REQUESTED** [review]
See the inline comments.

File: modelgen/src/main/java/cz/cvut/kbss/jopa/modelgen/ModelGenProcessor.java [hunk 4]
```diff
@@ -91,9 +91,6 @@ private boolean propertyIsWanted(Element param) {
         List<? extends AnnotationMirror> paramAnnotations = param.getAnnotationMirrors();
         if (!paramAnnotations.isEmpty()) {
             for (AnnotationMirror paramAnnotation : paramAnnotations) {
-                if (paramAnnotation.toString().contains("cz.cvut.kbss.jopa.model.annotations.Transient")) {
```
Comment (syrcheddar) [inline modelgen/src/main/java/cz/cvut/kbss/jopa/modelgen/ModelGenProcessor.java:94]: i'm not sure if this check is nessecary or not

**Review by ledsoft — APPROVED** [review]
LGTM

**Pull request merged**